components/suggest/src/store.rs

```diff
@@ -1387,6 +1439,20 @@ mod tests {
                 "icon": "3"
             }]),
         )?
+            .with_data(
+                "data-2.json",
+                json!([{
+                "description": "amo suggestion",
+                "url": "https://addons.mozilla.org/en-US/firefox/addon/example",
+                "guid": "{b9db16a4-6edc-47ec-a1f4-b86292ed211d}",
+                "keywords": ["relay", "spam", "masking", "alias"],
+                "title": "Firefox Relay",
+                "icon": "https://addons.mozilla.org/user-media/addon_icons/2633/2633704-64.png?modified=2c11a80b",
+                "rating": "4.9",
+                "number_of_ratings": 888,
+                "score": 0.25
+            },]),
```

Comment:
nitpick: `cargo fmt` doesn't (can't?) do a great job with macros, unfortunately—but could you please reformat this to fix the indentation in the `json!(...)`, closer to how we do `data-1.json` above?